### ISSUE TITLE
Bug: Problem in Footer after Logging In

### ISSUE THREAD

**Issue by ayushmangarg2003** [MilanCommunity/Milan#1070]
### Description

According to Code in Footer.jsx, after successful login their should be "EVENTS" instead of "Join Us" But this is not happening.

Here I have logged in :-
![Screenshot (40)](https://github.com/MilanCommunity/Milan/assets/105537793/bb1dea63-e140-421f-8bf1-76999b1aca84)

I don't think I can solve this because it is related to authToken I guess but I feel that this needs to be solved.

### Browser 🥦

Brave

### Checklist ✅

- [X] I checked and didn't find similar [issue](https://github.com/MilanCommunity/Milan/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/MilanCommunity/Milan/blob/main/CONTRIBUTING.md)
- [ ] I am willing to work on this issue (blank for no).

**Comment by tamalCodes** [comment]
That makes sense, thanks for raising the issue. @ayushmangarg2003

**Comment by tamalCodes** [comment]
Hi @venkadesh004 there are multiple things wrong with the PR as a result I had to close that. Please ask before raising the issue. Since this was related to cookies I needed to look this up first. Secondly please don't change the PR template.

You wrote `Issue No: #1070` , whereas in the template it was mentioned that the issue must be linked in the format of `Closes #1070`

Please make a new PR to the `beta` branch.

**Comment by YuvarajSingh-0** [comment]
Is this issue being resolved?

**Comment by tamalCodes** [comment]
@YuvarajSingh-0 , this is not being worked upon, would you like to work on it ?

**Comment by YuvarajSingh-0** [comment]
@tamalCodes sure.... But I need to discuss some things like... Data base access

**Comment by tamalCodes** [comment]
> @tamalCodes sure.... But I need to discuss some things like... Data base access

You don't need any kind of Database access, set up the frontend and the backend locally. You can spin up a local MongoDB connection which is free, and simply paste the URI in the `.env` of the Backend repo. Once the backend is up and running, start the Frontend too.

Once that is done make an account and signup. Look at the cookies section in the Application > Cookies tab, you will find a cookie called `isLoggedIn` which is set to be true. You can use that cookie directly in the Footer.

Use conditions like 
```jsx

{ Cookies.get("isLoggedIn") ? (Write the code) : (Write the code) }

**Comment by YuvarajSingh-0** [comment]
@tamalCodes I found that there are some typos with the token name here and there and there is also an issue with the backend side of setting cookie. While setting cookie, the httpOnly property is set to false and secure to true for both cookies to true, otherwise the cookie is not accessible in the code. So, I fixed it. 

There is also some issues with the logout it seems. Working on it.

Please tell me if there is something that bothers you. Thank You.

**Comment by YuvarajSingh-0** [comment]
Fixed the backend code also, how do I reference this issue in the backend repo?

**Comment by keshav-gupta19** [comment]
Is the Problem solved or may i look into it?
